src/Compatibility/Core/src/Android/Forms.cs

```diff
@@ -266,17 +271,19 @@ internal static void SendViewInitialized(this VisualElement self, global::Androi
 		}
 
 		static void SetupInit(
-			Context activity,
+			IMauiContext context,
 			Assembly resourceAssembly,
 			InitializationOptions? maybeOptions = null
 		)
 		{
+			var activity = (Activity)context.Context;
```

Comment:
Why do we cast as an Activity? Can it just stay as Context since we never use the type.